### PR TITLE
Add com.epicgames.ut99

### PR DESCRIPTION
**Pull request by AsciiWolf**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/OldUnreal/UnrealTournamentPatches/issues/1409
This Flatpak is just a wrapper around the official (binary-only) release of a community-made native port (included as extra-data).
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*
There is no upstream since it is an old, proprietary game. The OldUnreal community patch installation method on Linux is to manually copy its files over the original game directory, so desktop/AppStream files are not relevant. The game icon that I have used is a GPLv3 licensed one from this website: https://iconduck.com/icons/152633/unreal-tournament

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

File: com.epicgames.ut99.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+app-id: com.epicgames.ut99
+tags:
+  - proprietary
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: ut99
+finish-args:
+  - --socket=pulseaudio
+  - --socket=x11
```
Comment (AsciiWolf) [inline com.epicgames.ut99.yaml:10]: Does not work natively on Wayland. No game window is displayed when a wayland socket is enabled.

**Comment by pizzadude** [comment]
what about adding foxwsfix for ut99 like you did with ut2004? https://github.com/alexstrout/foxWSFix-UT99

File: com.epicgames.ut99.metainfo.xml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<component type="desktop-application">
+  <id>com.epicgames.ut99</id>
+  <launchable type="desktop-id">com.epicgames.ut99.desktop</launchable>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
```
Comment (bbhtt) [inline com.epicgames.ut99.metainfo.xml:6]: Add developer_name

Comment (AsciiWolf) [reply com.epicgames.ut99.metainfo.xml:6]: It already is [here](https://github.com/flathub/flathub/pull/4599/files#diff-eda253a221abd600482a95b3a8bc06da77f052fb55a91c477f5a07f4ff37d3fdR30). :-)

Comment (bbhtt) [reply com.epicgames.ut99.metainfo.xml:6]: 😅

**Comment by AsciiWolf** [comment]
> what about adding foxwsfix for ut99 like you did with ut2004? https://github.com/alexstrout/foxWSFix-UT99

Good suggestion, thanks! To be honest, I am not sure whether [foxWSFix](https://github.com/alexstrout/foxWSFix-UT99) is needed for OldUnreal. I need to test this first. :-)

**Comment by AsciiWolf** [comment]
bot, build com.epicgames.ut99

**Comment by AsciiWolf** [comment]
bot, build com.epicgames.ut99

**Comment by AsciiWolf** [comment]
I have added the foxWSFix support (and confirmed that it also gets enabled in ~/.utpg/System/UnrealTournament.ini), however I don't see any difference in the default HUD look.

**Comment by AsciiWolf** [comment]
I did more testing and decided to remove the foxWSFix support. It does not bring any visible changes (unlike in UT2004) and may cause compatibility issues with OldUnreal in the future since OldUnreal also brings some FOV fixes and plans to modify the default HUD layout (on widescreen displays) in the future.

**Comment by AsciiWolf** [comment]
bot, build com.epicgames.ut99

**Comment by AsciiWolf** [comment]
The last build did not finish yet, however the Flatpak is tested, works without issues and seems to be ready now. :-)

**Comment by pizzadude** [comment]
@AsciiWolf are you sure? because i definitely see a difference, maybe you need to put foxWSFix99.u in ~/.utpg/System also, that's what I did, but I'm not sure. but yeah if OldUnreal plans to have FOV fixes themselves maybe it's not worth it.

**Comment by AsciiWolf** [comment]
Yeah, it is possible that the file should be in `~/.utpg/System`, I did not test that. As far as I know, OldUnreal already has some FOV fixes implemented and there are plans to fix the HUD and other parts as well in future releases. Also, there are some reports like [this one](https://steamcommunity.com/app/13240/discussions/0/2993169283946758056/#c3765607455776521539) claiming that foxWSFix already caused bugs and crashes when used with OldUnreal. Given how often new OldUnreal versions are getting released, I am rather not risking possible compatibility issues.

**Pull request closed without merging**